com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/ScreenshotUtilities.cs

```diff
@@ -101,24 +101,15 @@ public static bool CaptureScreenshot(string path, int superSize = 1, bool transp
                 }
             }
 
-            // Create a camera clone with a transparent clear color.
+            // Create a camera clone.
             var renderCamera = new GameObject().AddComponent<Camera>();
+            renderCamera.CopyFrom(camera);

```

Comment:
Unrelated to outlines but had this tiny fix already in the works. Ensures screenshots have an opaque background when requested.